documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -95,7 +95,7 @@ Add them to your `persistence.xml` or where you put the rest of your Hibernate S
 
 Select Elasticsearch as the backend:: `hibernate.search.default.indexmanager elasticsearch`
 Hostname and port for Elasticsearch:: `hibernate.search.default.elasticsearch.host \http://127.0.0.1:9200`
-Selects the index creation strategy:: `hibernate.search.default.elasticsearch.index_schema_management_strategy CREATE_DELETE`
+Selects the index creation strategy:: `hibernate.search.default.elasticsearch.index_schema_management_strategy RECREATE_DELETE`
```

Comment:
I don't think it's a good idea to advertise `RECREATE_DELETE` in this example: it seems to be a bit risky - especially because other values looks sane. I would stick to the default for the example.